files/mc-start.sh

```diff
@@ -2,71 +2,6 @@
 
 set -euo pipefail
 
-export JAVA_OPTS_DEFAULT="-Dhazelcast.mc.home=${MC_DATA} -Djava.net.preferIPv4Stack=true"
-if [ -n "${JAVA_OPTS}" ]; then
-    export JAVA_OPTS="${JAVA_OPTS_DEFAULT} ${JAVA_OPTS}"
-else
-    export JAVA_OPTS="${JAVA_OPTS_DEFAULT}"
-fi
+cd "$(dirname "$0")/.."
```

Comment:
I'm wondering if this statement makes any sense at this point? To make it one step simpler you could just remove this and call `./hz-mc ...` on the following line.

Reply:
I've simplified it, what do you think?  https://github.com/hazelcast/management-center-openshift/pull/17/commits/6869d8a18187061f73ebf02469d0f6ffae2e923b